user-registration-app/src/app/views/user-registration-view/user-registration-view.component.ts

```diff
@@ -104,24 +167,35 @@ export class UserRegistrationViewComponent implements OnInit {
   businessChallengeRegister() {
     this.progressService.start();
     this.httpService.registerExtraCreditClass(
-      this.classes.find((c) => c.class_name === 'Business Challenge').uid.toString(),
+      this.classes.find(c => c.class_name === 'Business Challenge').uid.toString(),
     )
       .subscribe(() => {
         this.progressService.complete();
         this.alertsService.success('You are registered for the business challenge');
-      }, ({error}) => {
+      },         ({ error }) => {
         if (error.status === 409) {
           this.alertsService.success('You are registered for the business challenge');
         } else {
-          console.log(error);
           this.alertsService.warning('Something may have gone wrong in that process. Email technology@hackpsu.org or Contact a member of staff to check');
         }
       });
   }
 
   showClassName(class_name: string): boolean {
-    const hiddenClassNames = ['Business Challenge', 'CMPEN 362', 'STAT 463', 'SCM 421\n','EE 300W',
-    'CMPSC 131','STAT 463','IST 256','IST 210','STAT 380','CMPSC 311','CMPSC 442'];
-    return hiddenClassNames.indexOf(class_name) === -1;
+    // We don't remove classes from the DB, we just filter them here.
+    const shownClassNames = [
```

Comment:
I suspect this isnt working in staging because these classes are not in the staging db. I don't currently have the time to put them in and test but everything seems fine